### PR TITLE
[grafana] bump default grafana version 10.0.1

### PR DESCRIPTION
**Pull request by zzzinho**
update grafana default version to `v10.0.1`
[grafana v10.0.1 changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md#1001-2023-06-22)

### CODE REVIEW

**Comment by BobDu** [comment]
maybe we also need dump chat major version to: 7.0.0 ?

**Comment by zzzinho** [comment]
right! It should be 7.0.0 :)

**Review by brentheimer — APPROVED** [review]

File: charts/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
-version: 6.57.4
-appVersion: 9.5.5
+version: 7.0.0
+appVersion: 10.0.1
```
Comment (zanhsieh) [inline charts/grafana/Chart.yaml:4]: @zzzinho 
Thanks for the contribution. Can you specify what break change that the helm chart user should know, such as values.yaml change, upgrading / rollback note or strategy in README.md?

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/grafana/Chart.yaml (hunk 1 above)
Comment (rverenich) [reply charts/grafana/Chart.yaml:4]: @zzzinho , please use 6.x version for chart. There's no breaking changes, no need for "fancy" version number:

> A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.

**Comment by natl-set** [comment]
waiting for this!

File: charts/grafana/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
-version: 6.57.4
-appVersion: 9.5.5
+version: 9.6.0
+appVersion: 10.0.1
```
Comment (zanhsieh) [inline charts/grafana/Chart.yaml:4]: @zzzinho 
Please bump to:

```suggestion
version: 6.58.0
appVersion: 10.0.1
```

@rverenich is not one of the chart maintainers. You should not take his advice.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by rverenich — APPROVED** [review]

**Review by dbirks — APPROVED** [review]

File: charts/grafana/Chart.yaml (hunk 2 above)
Comment (zzzinho) [reply charts/grafana/Chart.yaml]: I committed suggestion :)

Comment (rverenich) [reply charts/grafana/Chart.yaml]: Hey, my advice was not to go with 7.0.0 chart version. Just bump 6.x chart version with `appVersion: 10.0.1`
@zzzinho , please approve and let's go further:) Thanks in advance

**Comment by LarsStegman** [comment]
@zanhsieh any chance this could be merged?

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**